Cargo.lock

```diff
@@ -1175,6 +1175,27 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "bindgen"
```

Comment:
Are all these new dependencies really needed?
bindgen, clang-sys, and rocksdb take a long time to compile, or require extra system dependencies.

Do we need to update the install documentation as well?

Reply:
We should not have rocksdb in our dependencies, looks like `default-features = false` is missing somewhere